packages/docusaurus-theme-classic/src/validateThemeConfig.ts

```diff
@@ -46,6 +46,7 @@ const DEFAULT_CONFIG = {
     minHeadingLevel: 2,
     maxHeadingLevel: 3,
   },
+  autoScrollTOC: true,
```

Comment:
We can keep this undocumented for now, probably not useful for most people, as their sites don't work because of broken CSS in the first place...